scripts/coe_stats_gui.lua

```diff
@@ -0,0 +1,90 @@
+---@class coe.StatsGUI
+local StatsGUI = {}
+
+local mod_gui = require( "mod-gui" )
+
+-- =============================================================================
+
+local function build_stats_info_frame( statistics_frame )
+  local info_frame = statistics_frame.add{
+    type = "frame",
+    name = "coe_stats_info_frame",
+    direction = "vertical"
+  }
+
+  info_frame.add{
+    type = "label",
+    caption = {"coe-stats-gui.label_launches_title"}
+  }.parent.add{
+    type = "flow",
+    direction = "horizontal"
+  }.add{
+    type = "label",
+    caption = {"coe-stats-gui.label_launches_required"}
+  }.parent.add{
+    type = "label",
+    caption = tostring( global.rocket_silo.required_launches )
+  }.parent.parent.add{
+    type = "flow",
+    direction = "horizontal"
+  }.add{
+    type = "label",
+    caption = {"coe-stats-gui.label_launches_complete"}
+  }.parent.add{
```

Comment:
Something to think about. Are the extra labels needed?
```txt
[coe-stats-gui]
label_launches_complete=Launches compete: __1__
```
```lua
caption = {"coe-stats-gui.label_launches_complete", global.rocket_silo.total_launches}
```

Reply:
I like the __1__ and will do that.
The required is needed because that number will change as the players die.  
It may be lost in chat or last known in a previous play session.  So I want it available there.